addons/udes_stock/models/stock_picking_print_strategy.py

```diff
@@ -23,3 +23,11 @@ def strategies(self, picking):
         return self.search([
             ('picking_type_id', '=', picking.picking_type_id.id),
         ])
+
+    @api.multi
+    def records(self, picking, report):
+        '''Return records to print for `report`.'''
+        picking.ensure_one()
+        if report.model == 'sale.order':
```

Comment:
Maybe pedantic.
This requires implicitly a dependency on odoo's `sale_stock` module, where `sale_line_id` is defined.
Wouldn't be cleaner to override this method in some customer module to add this `if` block?

Reply:
That was my original thought, but udes_stock has a dependancy on sale_stock in it's manifest so this should not be an issue.